.gitignore

```diff
@@ -162,7 +162,6 @@ cython_debug/
 .vscode
 dist
 DashAI.egg-info
-build
```

Comment:
I don't think it's a good idea to get builds from the gitignore.
The idea is that the repository does not contain any build file, but that the build is created outside.

Reply:
This build was redundant becuase there is already a ignore of build on line 10: https://github.com/DashAISoftware/DashAI/blob/c2da386cf4894fa8cc309f0c3ac4eb582a9e3f94/.gitignore#L10